list_of_standing_committees_and_working_groups.md

```diff
@@ -0,0 +1,27 @@
+# List of Standing Committees and Working Groups
+
+Project work that expands beyond software-related work is organized into chartered domains and is carried out by Standing Committees and Working Groups. A description of these bodies and their function can be found [here](standing_committees_and_working_groups.md). This document contains a list of the current Standing Committees and Working Groups with charter summaries and links to their GitHub repositories.
+
+## Standing Committees
+
+### Diversity, Equity, and Inclusion
+
+Charter Summary: Increase diversity, equity and inclusion within all aspects of Project Jupyter, including the various decision-making bodies/leadership, subprojects, and the broader community.
+
+### Code of Conduct Incident Response
+
+Charter Summary: Own and manage the Jupyter CoC reporting and incident response processes and make determinations on individual CoC incidents.
+
+### Conflict of Interest
+
+Charter Summary: Ensure that the joint and conflicting interests of leaders across Project Jupyter are disclosed appropriately, handled with transparency, and mitigated with integrity.
+
+## Working Groups
+
+### Trademark and Branding
+
+Charter Summary: License, protect, and promote the trademarks and visual and textual branding of Jupyter.
+
+## Community Building and Events
+
+[Charter](communitybuildingcommittee.md) Summary: Grow, build, and connect the global Jupyter community of users and contributors.
```

Comment:
Could we include the accessibility working group and a one-line charter summary in this document on the understanding that it'll be worked on by that group?
CC: @tonyfast, @isabela-pf

Reply:
Sure thing; thanks for the tag! Is there a specific timeline needed for the writing of a full charter?

```suggestion
[Charter](communitybuildingcommittee.md) Summary: Grow, build, and connect the global Jupyter community of users and contributors.

Charter summary: Build cross-project community, resources, and tools to support the inclusion of disabled people in Jupyter based on wider web accessibility open standards.

```

(@tonyfast let me know what you think of the summary.)

Reply:
suggestion suggestion:

> Charter Summary: Build cross-project community, resources, and tools to support the inclusion of disabled people in Jupyter <ins>applications and development</ins> <del>based on wider web accessibility open web accessibility standards</del>.

Reply:
Thank you both!

@isabela-pf I'm not sure about time-frame, we didn't have a specific date in mind, but since thre's already a regular meeting it seems apt to have it recognized as a working group going forward.

I'll let others weigh in if they think there's a date we should target.

Reply:
Agree, you can drive the timeline for this full charter. You can take a look at the charter for the Community Building Committee https://github.com/jupyter/governance/pull/105 and I will also email you the template draft I'm working on. Let me know if you have any questions or I can help with anything.